images/bigquery/Makefile

```diff
@@ -12,12 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = 0.2
+VERSION = 0.3
```

Comment:
Please use the standard `YYYYMMDD-commitish` pattern used elsewhere: https://github.com/kubernetes/test-infra/blob/master/jenkins/e2e-image/Makefile#L16

Reply:
done